### PR TITLE
[stable/atlantis] override configuration via configmap

### PR DESCRIPTION
**Pull request by anasinnyk**
Signed-off-by: Andrii Nasinnyk <anasinnyk@macpaw.com>

#### What this PR does / why we need it:
We have the possibility override any configuration via configmap. And we can use additional configuration options like `webhook`. (Right now we cannot use it via env variables)

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by sstarcher — APPROVED** [review]

**Comment by lkysow** [comment]
Hmm, I just left [this comment](https://github.com/helm/charts/pull/14796#issuecomment-507351149) on an identical PR 😄 

But I see now that for Slack there's really no other way to do it. I'm a bit busy right now but I'll hopefully get to this review soon. Thanks!

File: stable/atlantis/templates/configmap-config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{{- if .Values.config -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "atlantis.fullname" . }}
```
Comment (lkysow) [inline stable/atlantis/templates/configmap-config.yaml:5]: This shouldn't be called the name of the chart. It needs to be `-yaml-config` or something like that. Then we can have multiple config maps (we do already) and it's not confusing which each one is for.

Comment (anasinnyk) [reply stable/atlantis/templates/configmap-config.yaml:5]: Oh... sorry, I didn't think about that. It will be changing really easy.

**Pull request merged**